realhf/impl/model/interface/dpo_interface.py

```diff
@@ -181,17 +179,13 @@ def train_step(
         model.inc_version()
 
         res = {}
-        global_stats = constants.log_global_stats_tracker(
-            return_dict=True, clear_stats_after_logging=True
-        )
```

Comment:
Why remove? It there any bugs?

Reply:
It should not be removed. It was an accident.

Reply:
BTW I found that the logging of auxilary losses can be wrong with multiple mini-batches. I've marked it with FIXME in the code and will fix it in the future.